packages/amazon-vpc-cni-plugins/amazon-vpc-cni-plugins.spec

```diff
@@ -0,0 +1,100 @@
+%global vpccni_goproject github.com/aws
+%global vpccni_gorepo amazon-vpc-cni-plugins
+%global vpccni_goimport %{vpccni_goproject}/%{vpccni_gorepo}
+%global vpccni_gitrev be5214353252f8315a1341f4df9ffbd8cf69000c
+%global vpccni_gover 1.3
+
+# Construct reproducible tar archives
+# See https://reproducible-builds.org/docs/archives/
+%global source_date_epoch 1234567890
+%global tar_cf tar --sort=name --mtime="@%{source_date_epoch}" --owner=0 --group=0 --numeric-owner -cf
+
+Name: %{_cross_os}amazon-vpc-cni-plugins
+Version: %{vpccni_gover}
+Release: 1%{?dist}
+Summary: VPC CNI plugins for Amazon ECS and EKS
+License: Apache-2.0
+URL: https://%{vpccni_goimport}
+Source0: https://%{vpccni_goimport}/archive/%{vpccni_gitrev}/%{vpccni_gorepo}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+Requires: %{name}(binaries)
+
+%description
+%{summary}.
+
+%package bin
+Summary: VPC networking plugins binaries
+Provides: %{name}(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(fips) or %{name}-fips-bin)
+
+%description bin
+%{summary}.
+
+%package fips-bin
+Summary: VPC networking plugins binaries, FIPS edition
+Provides: %{name}(binaries)
+Requires: (%{_cross_os}image-feature(fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
+
+%description fips-bin
+%{summary}.
+
+%prep
+%autosetup -n %{vpccni_gorepo}-%{vpccni_gitrev}
+
+# Symlink amazon-vpc-cni-plugins-%{vpccni_gitrev} to the GOPATH location
+%cross_go_setup %{vpccni_gorepo}-%{vpccni_gitrev} %{vpccni_goproject} %{vpccni_goimport}
+
+%build
+# cross_go_configure cd's to the correct GOPATH location
+%cross_go_configure %{vpccni_goimport}
+LD_VPC_CNI_VERSION="-X github.com/aws/amazon-vpc-cni-plugins/version.Version=%{vpccni_gover}"
+VPC_CNI_HASH="%{vpccni_gitrev}"
+LD_VPC_CNI_SHORT_HASH="-X github.com/aws/amazon-vpc-cni-plugins/version.GitShortHash=${VPC_CNI_HASH::8}"
+
+declare -a VPC_CNI_BUILD_ARGS
+VPC_CNI_BUILD_ARGS=(
+  -ldflags "${GOLDFLAGS} ${LD_VPC_CNI_VERSION} ${LD_VPC_CNI_SHORT_HASH} ${LD_VPC_CNI_PORCELAIN}"
+)
+
+for p in \
```

Comment:
Lets build all the plugins under `plugins`:

https://github.com/aws/amazon-vpc-cni-plugins/tree/master/plugins

Reply:
Got it, will add the `ecs-serviceconnect`, `vpc-bridge`, and `vpc-tunnel` plugins here and also add symlinks to them in `ecs-agent`